testsuite/openshift/objects/auth_config/sections.py

```diff
@@ -197,27 +197,41 @@ def add_uma(self, name, endpoint, credentials_secret, **common_features):
 
 
 class ResponseSection(Section):
-    """Section which contains response configuration. todo envoyDynamicMetadata"""
+    """Section which contains response configuration."""
 
     @property
-    def success_headers(self):
-        """Nested dict for most of the section."""
-        return self.section.setdefault("success", {}).setdefault("headers", {})
+    def success(self):
+        """Nested dict for success dictionary."""
+        return self.section.setdefault("success", {})
 
     @modify
-    def clear_success_headers(self):
-        """Clears content of a success headers"""
+    def clear_success(self):
+        """Clears content of a success dictionary."""
         self.success_headers.clear()
 
+    @property
+    def success_headers(self):
+        """Nested dict for items wrapped as HTTP headers."""
+        return self.success.setdefault("headers", {})
+
+    @property
+    def success_dynamic_metadata(self):
+        """Nested dict for items wrapped as Envoy Dynamic Metadata."""
+        return self.success.setdefault("dynamicMetadata", {})
+
     def _add(
         self,
         name: str,
         value: dict,
+        wrapper: Literal["headers", "dynamicMetadata"] = "headers",
```

Comment:
`wrapper` is very important! Mention it in doc string (class or elsewhere, your call)

Reply:
Added documentation to `_add` function